chapters/Terminology.md

```diff
@@ -238,138 +261,104 @@ Schnorr
-  file) generated from the content itself such that for any given content and CID
-  generation algorithm there is a single, unique, collision-resistant identifier.
-  This is typically done through some hashing function.
-
-Content Identifiers
-
-: ::Content Identifier::
-
-CID
+Sidecar
 
-: ::Content Identifier::
+: A mechanism by which data necessary for resolving a DID is provided alongside
+  the **did:btc1** identifier being resolved, rather than being retrieved through
+  open and standardized means (e.g., by retrieving from IPFS).
 
-CIDs
+: To explain the metaphor, a sidecar on a motorcycle brings along a second passenger
+  in a transformed vehicle, the same way the DID controller MUST bring along the
+  DID document history to transform the situation into one that is verifiable.
 
-: ::Content Identifier::
+Sidecar Data
 
-Content Addressable Storage
+: Data transmitted via ::Sidecar::.
 
-: Content Addressable Storage (CAS) is a data storage system where content is
-  addressable using ::Content Identifiers:: (CIDs). The InterPlanetary File System
-  (IPFS) is an example of CAS.
 
-CAS
+Signal Blockheight
 
-: ::Content Addressable Storage::
+: The blockheight of the Bitcoin block that included a specific ::Beacon Signal::.
+  Blockheight is used as the internal time of the resolution algorithm.
 
-Non-Repudiation
+Singleton Beacon
 
-: Non-Repudiation is a feature of DID methods that can clearly state that all data
-  is available to present one canonical history for a DID.
+: A type of ::BTC1 Beacon:: whose ::Beacon Signals:: each contain a single ::BTC1 Update Announcement::.
 
-  If some data is needed but not available, the DID method MUST NOT allow DID
-  resolution to complete. Any changes to the history, such as may occur if a website
-  edits a file, MUST be detected and disallowed. The ::Late Publishing:: problem
-  breaks Non-Repudiation.
+Singleton Beacons
 
-Late Publishing
+: ::Singleton Beacon::
 
-: Late Publishing is the ability for DID updates to be revealed at a later point
-  in time, which alters the history of a DID document such that a state, that
-  appeared valid before the reveal, appears after Late Publishing to never have
-  been valid. Late Publishing breaks ::Non-Repudiation::.
+SMT Beacon
 
-Offline Creation
+: A type of ::BTC1 Beacon:: which aggregates multiple ::BTC1 Update Announcements::.  
+  A ::Beacon Signal:: from an SMT Beacon contains the root of an optimized ::Sparse Merkle Tree::.
 
-: Offline Creation refers to when a **did:btc1** identifier and corresponding
-  initial DID document are created without requiring network interactions.
+SMT Beacons
 
-**did:btc1** supports offline creation in two modes:
+: ::SMT Beacon::
 
-* Key Pair Deterministic Creation; and
-* DID Document Initiated Creation.
+Sparse Merkle Tree
 
-Sidecar
+: A ::Merkle Tree:: data structure where each data point included
+  at the leaf of the tree is indexed.
```

Comment:
This is a correct but incomplete description of a _sparse_ Merkle tree. I would add a definition for what makes it sparse:

```suggestion
: A ::Merkle Tree:: data structure where each data point included
  at the leaf of the tree is indexed. Empty leaves are discarded (zero values),
  making the tree sparse.
```

(Paraphrased from the alternative SMT appendix in #216.)

----

And it might not be appropriate to call attention to the optimizations (subtree collapsing etc.) in the terminology section. But if you wanted, you could also add something like this:

> Subtrees with a single path are collapsed to a single node, making the tree optimized.